thirdparty/freetype2/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ endif()
 ko_write_gitclone_script(
     GIT_CLONE_SCRIPT_FILENAME
     https://gitlab.com/koreader/freetype2.git
-    VER-2-10-2
+    VER-2-10-3
```

Comment:
Okay, changed cmakelists = this will be recompiled as will everything that depends on FT.